### PR TITLE
Link to new target

### PR DESCRIPTION
**Pull request by liubin**
The source may be moved to a separated repo from kubernetes.

### CODE REVIEW

File: contributors/devel/scheduler.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ scheduling policies to apply, and can add new ones.
 The policies that are applied when scheduling can be chosen in one of two ways.
 The default policies used are selected by the functions `defaultPredicates()` and `defaultPriorities()` in
 [pkg/scheduler/algorithmprovider/defaults/defaults.go](http://releases.k8s.io/HEAD/pkg/scheduler/algorithmprovider/defaults/defaults.go).
-However, the choice of policies can be overridden by passing the command-line flag `--policy-config-file` to the scheduler, pointing to a JSON file specifying which scheduling policies to use. See [examples/scheduler-policy-config.json](http://releases.k8s.io/HEAD/examples/scheduler-policy-config.json) for an example
+However, the choice of policies can be overridden by passing the command-line flag `--policy-config-file` to the scheduler, pointing to a JSON file specifying which scheduling policies to use. See [examples/scheduler-policy-config.json](https://github.com/kubernetes/examples/blob/master/staging/scheduler-policy-config.json) for an example
```
Comment (cblecker) [inline contributors/devel/scheduler.md]: Please use https://git.k8s.io/examples/staging/scheduler-policy-config.json

**Comment by liubin** [comment]
Hi @cblecker have changed the link, please check it again, thanks.

**Pull request merged**